### PR TITLE
Add permanent limp quirk

### PR DESCRIPTION
**Pull request by TheGreatKitsune**
## About The Pull Request
Adds a permanent limp quirk, negative with a cost of -3. Allows adding a minor, moderate, or major limp to a characters leg that cannot be treated short of limb replacement.
## Why It's Good For The Game
Requested by a player. Allows for players to have a character that limps naturally without having to dislocate a leg in round.

## Proof Of Testing
<details>
<summary>Screenshots/Videos</summary>

perm_limp_1
perm_limp_2
perm_limp_3

</details>

## Changelog
:cl: Cerami
add: Adds a new negative quirk: permanent limp
/:cl:

Edit:
Changed cost to -3
Made incompatible with paraplegic and hemiplegic

### CODE REVIEW

**Comment by ReturnToZender** [comment]
I think this would at least be a -2 trait, considering it severely impairs your movement

**Comment by YakumoChen** [comment]
This sounds like something upstream tg may like as an aside

**Comment by Doom-external** [comment]
Dr. Gregory House

**Comment by TheGreatKitsune** [comment]
Admittedly I don't have much idea what sort of point value would be good. I might also look into how to prevent stacking this quirk with paraplegic perhaps as well

**Comment by ReturnToZender** [comment]
> Admittedly I don't have much idea what sort of point value would be good. I might also look into how to prevent stacking this quirk with paraplegic perhaps as well

There's a glob list of mutually exclusive quirks that you can add this to

File: code/datums/wounds/permanent_limp.dm [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+	require_any_biostate = TRUE
+
+	required_wounding_types = list(WOUND_ALL)
+
+	wound_series = WOUND_SERIES_PERM_LIMP_BASIC
+
+/datum/wound_pregen_data/perm_limp/right
+	abstract = FALSE
+	wound_path_to_generate = /datum/wound/perm_limp/right
+/datum/wound_pregen_data/perm_limp/right/moderate
```
Comment (Arturlang) [inline code/datums/wounds/permanent_limp.dm:49]: Put empty spaces before type definitions

**Review by Arturlang — CHANGES_REQUESTED** [review]

**Pull request merged**